mlflow/keras.py

```diff
@@ -89,7 +89,15 @@ def save_model(keras_model, path, conda_env=None, mlflow_model=Model()):
     os.makedirs(path)
     model_data_subpath = "model.h5"
     keras_model.save(os.path.join(path, model_data_subpath))
-
+    if custom_objects is not None:
```

Comment:
If `custom_objects` is specified, we should also add `cloudpickle` as a dependency of the MLflow model. As a reference, the `mlflow.sklearn` module conditionally adds this dependency (see https://github.com/mlflow/mlflow/blob/26547aa98cc124ecdf8c5bd8cf80a6358da589b1/mlflow/sklearn.py#L129-L130).

We can achieve backwards compatibility when adding a parameter to the `mlflow.keras.get_default_conda_env()` method by using a keyword argument with a default value of `False` (reference: https://github.com/mlflow/mlflow/blob/26547aa98cc124ecdf8c5bd8cf80a6358da589b1/mlflow/sklearn.py#L39)